template/page-events.html.jinja

```diff
@@ -328,6 +328,26 @@
     "image": "AMAs.png",
     "end_date": "March 15, 2023"
     },
+    {
+    "name": "XRPL Community Meetup: Madrid",      
+    "description": "We're excited to host our second Miami meetup for XRP Ledger community members on April 6th from 6-8pm, featuring Marco Neri, Developer Advocate at Ripple, who will join us to give a presentation on the XRP Ledger.",
+    "type": "meetup",
+    "link": "https://www.meetup.com/xrp-ledger-espana-madrid-y-barcelona/events/292597878",
+    "location": "Madrid",
+    "date": "April 29, 2023",
+    "image": "Madrid.png",
+    "end_date": "April 29, 2023" 
```

Comment:
The dates seem to be having some discrepancy?

> our second Miami meetup for XRP Ledger community members on **April 6th**

>    "location": "Madrid",
    "date": "**April 29, 2023**",